Dockerfile

```diff
@@ -16,7 +16,7 @@ COPY --chown=node:node package.json .
 COPY --chown=node:node yarn.lock .
 
 # Install the dependencies
-RUN yarn install
+RUN yarn install --frozen-lockfile
```

Comment:
why are we freezing the lock file?

Reply:
This prevents the edge case of a lock file being changed only inside a container and not on the host machine, which could in theory cause a hard to trace bug, albeit this is not likely.